posts/2023-08-03-23.0.0.8-beta.adoc

```diff
@@ -0,0 +1,142 @@
+// // // // // // // // 
+[#sp3]
+== Spring Boot 3.0 Support
+
+The Liberty Spring boot 3.0 feature provides a more complete support for running a Sprint Boot 3.0 application on Liberty as well as having the capability to thin the application when creating applications in containers. 
+
+Prior releases of Liberty provided support for Spring Boot 1.5 and Spring Boot 2.0 applications.  In the 23.0.0.3 release, support was extended to include link:/blog/2023/06/15/running-spring-boot-3.html[Running a Spring Boot 3.x application WAR file on Liberty].  With the creation of this new Spring Boot 3.0 feature, users will now have support for both `.wars` and `.jars` by enabling the `springboot-3.0` feature.
+
+To utilize the Spring Boot 3.x functionality, you must be running with Java 17 or higher.  Additionally, if your application makes use of the link:/docs/latest/reference/feature/servlet.html[Jakarta Servlet] feature, it must be version 6.0.  These features would be configured in the `server.xml` file as follows:
+
+[source,xml]
+----
+<features>
+   <feature>springboot-3.0</feature>
+   <feature>servlet-6.0</feature>
+</features>
+----
+
+Additionally, similarly to previous feature versions, link:/docs/latest/reference/config/springBootApplication.html[Spring Boot configuration elements] can be specified via the `server.xml` file. For example:
+
+[source,xml]
+----
+ <springBootApplication id="spring-boot-app" location="spring-boot-app-0.1.0.jar" name="spring-boot-app" />
+----
+
+A few migration notes for using Spring Boot 3:
+
+* link:/docs/latest/reference/feature/webProfile-10.0.html[Jakarta EE 10 features] must be used. 
+
+* Security configuration requires in-application modifications. 
+  - link:https://spring.io/blog/2022/02/21/spring-security-without-the-websecurityconfigureradapter[Spring Security without the WebSecurityConfigurerAdapter]
+
+  - link:https://docs.spring.io/spring-security/reference/servlet/authorization/authorize-http-requests.html[Authorize HttpServletRequests]
+
+  - link:https://stackoverflow.com/questions/72381114/spring-security-upgrading-the-deprecated-websecurityconfigureradapter-in-spring[Spring Security: Upgrading the deprecated WebSecurityConfigurerAdapter in Spring Boot 2.7.0]
+
+* A new server template (`springBoot3`) is available.
+
+
+
+   
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC> 
+
+
+[#data]
+== Early Preview of Jakarta Data
+
+The 23.0.0.8-beta release also includes a preview of Jakarta Data.  More information is available in the link:{url-prefix}/blog/2023/02/21/23.0.0.7-beta.html[Jakarta Data beta 2 update in Open Liberty 23.0.0.7-beta] blog post.
+
+
+[#run]
+=== Try it now 
+
+To try out these features, update your build tools to pull the Open Liberty All Beta Features package instead of the main release. The beta works with Java SE 20, Java SE 17, Java SE 11, and Java SE 8.
+
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], you can install the All Beta Features package using:
```

Comment:
```suggestion
If you're using link:{url-prefix}/guides/maven-intro.html[Maven], add the All Beta Features package to your `pom.xml` file:
```

Reply:
Note that if we want this wording moving forward, it should be updated in the template